httpapi/compose.go

```diff
@@ -49,7 +49,7 @@ func Compose(apis ...API) http.Handler {
 
 type oobKey string
 
-const OOB_KEY oobKey = "gobits-httpapi-oob"
+const OOBKey oobKey = "gobits-httpapi-oob"
```

Comment:
Since we don't intend to export it, it should probably be lower-cased. To avoid a name collision with the type, I suggest:
```suggestion
const oobFunctionKey oobKey = "gobits-httpapi-oob"
```